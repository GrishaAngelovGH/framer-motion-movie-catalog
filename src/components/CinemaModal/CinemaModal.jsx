const CinemaModal = ({ onClose }) => (
  <div className="fixed top-0 bg-black bg-opacity-75 w-screen h-screen flex justify-center items-center">
    <div className="bg-white w-[320px] md:w-3/4 xl:w-1/2">
      <div className="flex justify-between items-center bg-gray-100 shadow-lg p-3">
        <h1 className="text-2xl text-center">Available Cinemas</h1>

        <svg onClick={onClose} xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="size-6 hover:bg-gray-400 hover:text-white hover:rounded-full hover:p-1 hover:cursor-pointer">
          <path strokeLinecap="round" strokeLinejoin="round" d="M6 18 18 6M6 6l12 12" />
        </svg>
      </div>
      <div className="max-h-[300px] overflow-auto">
      </div>
    </div>
  </div>
)

export default CinemaModal