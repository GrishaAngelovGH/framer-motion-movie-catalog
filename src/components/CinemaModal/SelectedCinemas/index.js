import SelectedCinemas from "./SelectedCinemas"

export default SelectedCinemas