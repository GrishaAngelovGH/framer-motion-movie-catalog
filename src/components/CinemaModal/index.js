import CinemaModal from "./CinemaModal"

export default CinemaModal