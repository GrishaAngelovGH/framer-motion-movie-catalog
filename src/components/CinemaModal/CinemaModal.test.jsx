import { render } from "@testing-library/react"

import CinemaModal from "./CinemaModal"

test("should render CinemaModal component", () => {
  const view = render(<CinemaModal />
  )

  expect(view).toMatchSnapshot()
})