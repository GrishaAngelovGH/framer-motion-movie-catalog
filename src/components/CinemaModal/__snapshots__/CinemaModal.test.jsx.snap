// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render CinemaModal component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="fixed top-0 bg-black bg-opacity-75 w-screen h-screen flex justify-center items-center"
      >
        <div
          class="bg-white w-[320px] md:w-3/4 xl:w-1/2"
        >
          <div
            class="flex justify-between items-center bg-gray-100 shadow-lg p-3"
          >
            <h1
              class="text-2xl text-center"
            >
              Available Cinemas
            </h1>
            <svg
              class="size-6 hover:bg-gray-400 hover:text-white hover:rounded-full hover:p-1 hover:cursor-pointer"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M6 18 18 6M6 6l12 12"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </div>
          <div
            class="max-h-[300px] overflow-auto"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="fixed top-0 bg-black bg-opacity-75 w-screen h-screen flex justify-center items-center"
    >
      <div
        class="bg-white w-[320px] md:w-3/4 xl:w-1/2"
      >
        <div
          class="flex justify-between items-center bg-gray-100 shadow-lg p-3"
        >
          <h1
            class="text-2xl text-center"
          >
            Available Cinemas
          </h1>
          <svg
            class="size-6 hover:bg-gray-400 hover:text-white hover:rounded-full hover:p-1 hover:cursor-pointer"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M6 18 18 6M6 6l12 12"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </div>
        <div
          class="max-h-[300px] overflow-auto"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
